### PR TITLE
initial commit for auto publicize config tool

### PR DESCRIPTION
**Pull request by droslean**
This tool re-generates the publicize plugin configuration file and submits a pull request against the `openshift/release` repository.

/cc @openshift/openshift-team-developer-productivity-test-platform  

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: cmd/autopublicizeconfig/main.go [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+		labelsToAdd = append(labelsToAdd, labels.Approved, labels.LGTM)
+	}
+
+	source := fmt.Sprintf("%s:%s", o.githubLogin, remoteBranch)
+	if err := bumper.UpdatePullRequestWithLabels(gc, githubOrg, githubRepo, title, description, matchTitle, source, "master", true, labelsToAdd); err != nil {
+		logrus.WithError(err).Fatal("PR creation failed.")
+	}
+}
+
+func getReposForPrivateOrg(releaseRepoPath string, whitelist map[string][]string) (map[string]sets.String, error) {
```
Comment (alvaroaleman) [inline cmd/autopublicizeconfig/main.go:175]: s/whitelist/allowlist/g

Comment (droslean) [reply cmd/autopublicizeconfig/main.go:175]: this is a term that we are already using. see also https://github.com/openshift/ci-tools/blob/master/pkg/config/whitelist.go

Comment (alvaroaleman) [reply cmd/autopublicizeconfig/main.go:175]: https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language

Feel free to also change the terminology in that package (in a different PR)

Comment (droslean) [reply cmd/autopublicizeconfig/main.go:175]: Oh!!! Thanks for letting me know. I will make the changes

**Pull request merged**